vexl-it/vexl
# feat: #106 - Reupload offers that do not exist on server

How to test: 

1. Create a few offers
2. Your contacts should be able to see them 
3. Go to developer screen (by pressing version number text 6x) and press on `simulate offers deleted from server`. Wait for done alert to appear.
4. Your contacts should not be able to see your offers
5. switch to different app and than back to Vexl
6. You should see Vexlbot warning on marketplace and on myOffers list
7. After pressing on `reencrypt offers` offers should start reencrypting
8. After offers are reencrypted others should be able to see them again.
